docs/dev/Lua API.rst

```diff
@@ -1679,21 +1695,45 @@ Units module
 
   Finds (or creates if requested) a misc trait object with the given id.
 
+* ``dfhack.units.getRaceNameById(race_id)``
+* ``dfhack.units.getRaceName(unit)``
+
+  Get raw token name (e.g., "DWARF").
+
+* ``dfhack.units.getRaceReadableNameById(race_id)``
+* ``dfhack.units.getRaceReadableName(unit)``
+* ``dfhack.units.getRaceNamePluralById(race_id)``
+* ``dfhack.units.getRaceNamePlural(unit)``
+
+  Get human-readable name (e.g., "dwarf" or "dwarves").
+
+* ``dfhack.units.getRaceBabyNameById(race_id[,plural])``
+* ``dfhack.units.getRaceBabyName(unit[,plural])``
+* ``dfhack.units.getRaceChildNameById(race_id[,plural])``
+* ``dfhack.units.getRaceChildName(unit[,plural])``
+
+  Get human-readable baby or child name (e.g., "dwarven baby" or "dwarven child").
+
+* ``dfhack.units.getReadableName(unit)``
```

Comment:
sorry about this, but expect a few merge conflicts with https://github.com/DFHack/dfhack/pull/4816